### ISSUE TITLE
list in parameters section

### ISSUE THREAD

**Issue by canicola** [numpy/numpydoc#113]
I have the following question. Is it possible to allow in a Parameters section (or Returns etc) to have a list? I would like to have something like:

```
        """
        Parameters
        ----------
        param 1 : pandas data frame
            This would be really cool to allow the following list and make it more readable:

            * **index:** Array-like, integer valued representing
              days. Has to be sorted and increasing.
            * **dtype:** float64. Value of temperature.
            * **columns:** location description, e.g. 'San Diego'

        param 2 : int
            nice number!
        """

``` 
However the output looks odd. See the first picture. The font is too big and it doesnt' treat it as text for param 1 which means there is no text next to the "-".  Is there a way to achieve this? 

I checked the source and it looks that bullet list are supported. I'm just not sure if this work within the Parameters / Attributes / Return sections.

If I'm using no blank line before the list:

```
        """
        Parameters
        ----------
        param 1 : pandas data frame
            This would be really cool to allow the following list and make it more readable:
             * **index:** Array-like, integer valued representing
               days. Has to be sorted and increasing.
             * **dtype:** float64. Value of temperature.
             * **columns:** location description, e.g. 'San Diego'

        param 2 : int
            nice number!
        """
```

I get correct fonts but it's now bold (see second picture) and again the text doesn't start after the "-"

FIRST VERSION
![screenshot from 2017-10-05 14-01-38](https://user-images.githubusercontent.com/8946183/31226566-de09203e-a9d6-11e7-83cd-36784b22a226.png)

SECOND VERSION (WITHOUT BLANK LINE)
![screenshot from 2017-10-05 14-09-14](https://user-images.githubusercontent.com/8946183/31226551-cb3e4722-a9d6-11e7-81fc-89e8148dec32.png)

**Comment by jnothman** [comment]
I am quite confused about that rendering. Why are you getting bullet points with text like "1 (param)"? I don't think there i should be bullets at all.

**Comment by canicola** [comment]
that's fine. It treats the 1 as name which makes sense since I've used parm 1 : int. I.e. the name of the first parameter is treated as "1". The original problem still is the confusing one.

**Comment by jnothman** [comment]
No, I suspect they are related. It's being parsed differently to usual.

**Comment by jnothman** [comment]
I can't reproduce any of:
* your bullet points
* your rendering quirks
* the difference between having that blank line there or not.

Are you sure the `numpydoc` extension is in your Sphinx `conf.py`?

**Comment by jnothman** [comment]
No problem. Glad it went away!
